core/src/edu/asu/c3simulator/screens/BusinessCreationGuide.java

```diff
@@ -0,0 +1,218 @@
+package edu.asu.c3simulator.screens;
+
+import com.badlogic.gdx.Game;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.Screen;
+import com.badlogic.gdx.graphics.GL20;
+import com.badlogic.gdx.scenes.scene2d.InputEvent;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+import com.badlogic.gdx.scenes.scene2d.ui.TextButton;
+import com.badlogic.gdx.scenes.scene2d.utils.ClickListener;
+
+/**
+ * Display a panel consisting of steps necessary to create a business and navigates
+ * through them. Includes a button to continue on to the next step.
+ * 
+ * @author Alyahya, Mohammed
+ * 
+ */
+public class BusinessCreationGuide implements Screen
+{
+	private class navigationButtonListener extends ClickListener
+	{
+		BusinessCreationGuideScreens targetedScreen;
+		String screenName;
+		
+		/**
+		 * @param targetedScreen
+		 *            The screen which the button will show.
+		 * @param screenName
+		 *            The name of the screen.
+		 */
+		public navigationButtonListener(BusinessCreationGuideScreens targetedScreen,
+				String screenName)
+		{
+			this.targetedScreen = targetedScreen;
+			this.screenName = screenName;
+		}
+		
+		@Override
+		public void clicked(InputEvent event, float x, float y)
+		{
+			System.out.println(screenName);
+			showScreen(targetedScreen);
+		}
+	}
+	
+	private class progressButtonListener extends ClickListener
+	{
+		@Override
+		public void clicked(InputEvent event, float x, float y)
+		{
+			if (currentScreen == industryScreen)
+			{
+				System.out.println("Transitioning to Direction Screen");
+				showScreen(directionScreen);
+			}
+			else if (currentScreen == directionScreen)
+			{
+				System.out.println("Transitioning to funding Screen");
+				showScreen(fundingScreen);
+			}
+			else if (currentScreen == fundingScreen)
+			{
+				System.out.println("Transitioning to tasks Screen");
+				showScreen(tasksScreen);
+			}
+			else if (currentScreen == tasksScreen)
+			{
+				System.out.println("Transitioning to Home Screen");
+				// TODO: Transition to main hub
+				// TODO: pass user choices to data layer
+			}
+		}
+	}
```

Comment:
Tagged for refactoring. See above.

Also add a task to the backlog for this (not the sprint backlog)